clients/client-core/src/client/real_messages_control/real_traffic_stream.rs

```diff
@@ -44,6 +48,15 @@ const MAX_DELAY_MULTIPLIER: u32 = 6;
 // The minium multiplier we apply to the base average Poisson delay.
 const MIN_DELAY_MULTIPLIER: u32 = 1;
 
+// When prioritizing traffic it can be used to divide lanes into new and old.
+const MSG_CONSIDERED_OLD_AFTER_SECS: u64 = 5;
```

Comment:
should those perhaps go to our config file so that we could mess with them more easily?

Reply:
Agree! I'll add this to the todo for the next PR